libmachine/drivers/plugin/localbinary/utils_nolinux.go

```diff
@@ -0,0 +1,11 @@
+//go:build !linux
```

Comment:
These should be 
```
//go:build windows
// +build windows
```
to match how other repositories handle multiple platforms

Reply:
Also, the file should have a suffix `_windows.go`, to match how other repositories handle this case.

Reply:
fixed. 

The other file is renamed to `utils.go` as it is for every platform except windows